docs/source/system_reference_guide/jobsui.ipynb

```diff
@@ -62,10 +62,10 @@
     "\n",
     "Navigating to the product directory for the selected job above shows the following:\n",
     "```\n",
-    "(base) root@workspacelbx1uqgeyqdei83w:~/my-private-bucket/dps_output/run-dps-test_ubuntu/delay10/2023/05/10/15/13/27/250000# pwd\n",
+    "(vanilla) root@workspacelbx1uqgeyqdei83w:~/my-private-bucket/dps_output/run-dps-test_ubuntu/delay10/2023/05/10/15/13/27/250000# pwd\n",
```

Comment:
Do we want to just change this to `pangeo` because `vanilla` is potentially getting renamed to `python` in the next release? https://docs.google.com/spreadsheets/d/16DIRmgBjWIMc6kWY7W2jEwtsfdruQtg2ygb8oELLaW4/edit#gid=0

Reply:
just committed this @grallewellyn

Reply:
Thank you!